### ISSUE TITLE
Problem with certificates on windows only?

### ISSUE THREAD

**Issue by norbe** [Kdyby/CurlCaBundle#1]
Hi,

I just discover problem that on windows machines I'm not able to crawl some sites (for example http://ssl.heureka.cz/login-process/). Even when I use CurlCaBundle I still getting error SSL Certificate problem: Unable to get local issuer certificate.

The same domain is used for Heureka Kosik Api and it works ok (on production server).

When I use quite old crt file from http://filehostuk.com/downloads/cacert.rar it works ok (i found it thanx to [this post](https://www.zen-cart.com/showthread.php?213892-Windows-server-Curl-error-%2860%29-SSL-Certificate-problem-Unable-to-get-local-issuer&s=c5a187f81b0e1437b6c8f3bf43fbf81a&p=1284548#post1284548)). Do you have any idea why?

**Comment by fprochazka** [comment]
Interesting, I'll try to upgrade the script, haven't done that in a while, maybe it will fix the problem.
Otherwise I think I'll have to setup testing, if the certificates work on buch of domains.

**Comment by fprochazka** [comment]
I've already have to revert one cert for our usage https://github.com/Kdyby/CurlCaBundle/tree/with-old-cybertrust-root.
Maybe it's related to mozilla droping the "weak" certs? http://curl.haxx.se/docs/caextract.html

**Comment by fprochazka** [comment]
https://www.ssllabs.com/ssltest/analyze.html?d=heureka.cz&latest "Certificate uses a weak signature. When renewing, ensure you upgrade to SHA2."

**Comment by norbe** [comment]
It's hard to say as I don't know how it is processed internally in curl. Is it possible that on linux it's enought that top level authority is trusted and on windows it requires all levels to be trusted?

When I compare ca-bundle.crt on my dev machine with that from production server, there is no change in "thawte" brand (and even if I replace my ca-bundle.crt with that from production it do not help)...

**Comment by fprochazka** [comment]
I'm guessing the problem started with this commit, right? https://github.com/Kdyby/CurlCaBundle/commit/50b9eefd2ef9b48075408a9752732af8af56b472

**Comment by norbe** [comment]
Yes, when I try ca-bundle.crt before that commit, it works. After that it ends with SSL error...

**Comment by fprochazka** [comment]
You can just lock on the previous tag [v1.0.5](https://github.com/Kdyby/CurlCaBundle/releases/tag/v1.0.5) and it should work till we figure something out...
I'm guessing I could create another branch that would contain the weaker certs.

**Comment by norbe** [comment]
Ok, thanks
